src/libs/actions/IOU.ts

```diff
@@ -2271,25 +2272,25 @@ function createSplitsAndOnyxData(
     if (!existingSplitChatReport) {
         existingSplitChatReport = participants.length < 2 ? ReportUtils.getChatByParticipants(participantAccountIDs) : null;
     }
-    let newChat: ReportUtils.OptimisticChatReport = {};
-
-    if (!existingSplitChatReport) {
-        if (participants.length > 1) {
-            newChat = ReportUtils.buildOptimisticChatReport(
-                participantAccountIDs,
-                '',
-                CONST.REPORT.CHAT_TYPE.GROUP,
-                undefined,
-                undefined,
-                undefined,
-                undefined,
-                undefined,
-                undefined,
-                CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN,
-            );
-        } else {
-            newChat = ReportUtils.buildOptimisticChatReport(participantAccountIDs);
-        }
+    let newChat: ReportUtils.OptimisticChatReport | EmptyObject = {};
+
+    if (!existingSplitChatReport && participants.length > 1) {
+        const reportName = GroupChatUtils.getGroupChatName(participantAccountIDs);
```

Comment:
Hmm we should have them in there for now.

This is part of why we are switching to `participants` instead of the `participantAccountIDs`. Do we have the full `participants` here? If so, maybe we can use that instead for the case of a "Group Chat".